README.md

```diff
@@ -1,4 +1,5 @@
-# Airship
+# ![](airship.svg) Airship 
```

Comment:
What about removing `There are three npm packages for easier use:` or changing it to make it clearer?

The logo is too big compared to the text. What about using a `<img>` tag and set the height to the font size?